openocd/stm32h7x_spiflash.cfg

```diff
@@ -63,8 +63,14 @@ proc hxa-001_qspi_init { } {
 	sleep 20								;# wait for the flash to come out of reset
 
 	mmw 0x52005000 0x30000000 0x00000001	;# OCTOSPI_CR |= FMODE=0x3, &= ~EN
-	# OCTOSPI1: memory-mapped 1-line read mode with 3-byte addresses
-	mww 0x52005100 0x01002101				;# OCTOSPI_CCR: DMODE=0x1, ABMODE=0x0, ADSIZE=0x2, ADMODE=0x1, ISIZE=0x0, IMODE=0x1
+
+	if { [env LARGE_FLASH] == 0 } {
```

Comment:
Is this branch taken when LARGE_FLASH is unset, i.e. not set to "0"? Just want to be sure :)

Reply:
That would be my understanding yes. But I could flip it to make it more clear and explicit and be inline with the check in ./5_restore.sh. I'll do that just so the check align.

Reply:
I checked and double checked.

*  [env LARGE_FLASH] actually ONLY checks if LARGE_FLASH is present in the environment variables

I have updated the code to both test for presents and for the value.

So you did actually dodge a bullet here. 
Thanks for the review.